apps/web-giddh/src/app/reports/components/report-details-components/report.details.component.ts

```diff
@@ -109,8 +111,12 @@ export class ReportsDetailsComponent implements OnInit, OnDestroy {
         private companyService: CompanyService,
         private _toaster: ToasterService,
         private settingsBranchAction: SettingsBranchActions,
-        private generalService: GeneralService) {
-
+        private generalService: GeneralService, private breakPointObservar: BreakpointObserver) {
```

Comment:
Always add new dependencies in constructor on a new line
```suggestion
        private generalService: GeneralService,
        private breakPointObservar: BreakpointObserver) {
```